torch_disc/torch_disc/csrc/disc_compiler/passes/cluster.cpp

```diff
@@ -24,36 +24,38 @@ using namespace ::torch::jit;
 // conversion module into a group. We should re-implement this.
 std::vector<Node*> FakeCluster(const std::shared_ptr<Graph>& graph) {
   std::vector<Node*> nodes;
-  for (auto node : graph->nodes()) {
-    if (torch::blade::IsMlirMhloSupported(*node) &&
-        node->kind() != prim::Constant) {
-      if (node->kind() == aten::addmm) continue;
-      nodes.push_back(node);
-    }
-  }
+  std::copy_if(graph->nodes().begin(), graph->nodes().end(),
```

Comment:
`cbegin`

Reply:
may not, we need the mutable node pointer, because it would be destroyed later.

https://github.com/alibaba/BladeDISC/blob/41dfc36902cf63c0e488c970da7b8a4e7763ce9a/torch_disc/torch_disc/csrc/disc_compiler/passes/register_disc_class.cpp#L99